agent-inject/handler.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/hashicorp/vault-k8s/agent-inject/agent"
 	"github.com/hashicorp/vault/sdk/helper/strutil"
 	"github.com/mattbaird/jsonpatch"
-	"k8s.io/api/admission/v1beta1"
+	v1 "k8s.io/api/admission/v1"
```

Comment:
nit: The alias seems unnecessary as-is, but should this be `admissionv1` to follow the example of other similar packages?

Reply:
Yep, that does look odd. Fixed in fe858b700c4f856183c027c91a8cdaca8b798f3e.